examples/django_chat/chat/sockets.py

```diff
@@ -0,0 +1,64 @@
+import logging
+from sdjango import namespace
+
+
+online_user_num = 0
+
+
+@namespace('/test')
+class TestNamespace:
+
+    def __init__(self, name):
+        self.name = name
+        self.request = None  # django request object
+
+    def _get_socket(self, sid):
+        socket = namespace.server.eio._get_socket(sid)
```

Comment:
The `eio._get_socket()` method is private. The `_` prefix indicates it is not meant to be called from the outside. Is there any way you can do what you need to do without using the internal socket object? I really don't like the idea of promoting its use in an example application.